index.html

```diff
@@ -1,179 +1,160 @@
+        <div class="article__content">
+          <h3 class="article__title">
+            <a class="article__title-link" href="#">Join us for our Spring fundraisers</a>
+          </h3>
+          <p class="article__summary">
+            Riders needed now to help raise awareness and funds for BfR and
+            Médecins Sans Frontières.
+          </p>
         </div>
-
-        <div class="sidebar">
-          <h2 class="heading-underline">Upcoming events</h2>
-
-          <div class="article">
-            <img
-              class="article__thumbnail"
-              src="spring-fundraisers_thumbnail.jpg"
-              alt=""
-            />
-            <div class="article__content">
-              <h3 class="article__title">
-                <a class="article__title-link" href="#">Join us for our Spring fundraisers</a>
-              </h3>
-              <p class="article__summary">
-                Riders needed now to help raise awareness and funds for BfR and
-                Médecins Sans Frontières.
-              </p>
-            </div>
-          </div>
-
-          <div class="article">
-            <img
-              class="article__thumbnail"
-              src="bikes-for-refugees_logo.jpg"
-              alt=""
-            />
-            <div class="article__content">
-              <h3 class="article__title">
-                <a class="article__title-link" href="#">Bike dropoff event in Edinburgh on May 11</a>
-              </h3>
-            </div>
-          </div>
-
-          <div class="article">
-            <img
-              class="article__thumbnail"
-              src="edinburgh-damascus_thumbnail.png"
-              alt=""
-            />
-            <div class="article__content">
-              <h3 class="article__title">
-                <a class="article__title-link" href="#">Help us cycle 4,797km</a>
-              </h3>
-              <p class="article__summary">
-                We need you to help us cycle 4797km, the distance of Edinburgh
-                to Damascus.
-              </p>
-            </div>
-          </div>
-
-          <a
-            href="https://www.facebook.com/BikesforRefugeesScotland/events/?ref=page_internal"
-            target="_blank"
-            class="facebook-link"
-          >
-            <span class="fa fa-facebook-square"></span>
-            All Facebook events
-          </a>
+      </article>
+
+      <article class="article">
+        <img class="article__thumbnail" src="images/bikes-for-refugees_logo.jpg" alt="Bike logo" />
+        <div class="article__content">
+          <h3 class="article__title">
+            <a class="article__title-link" href="#">Bike dropoff event in Edinburgh on May 11</a>
+          </h3>
         </div>
-    </div>
+      </article>
+
+      <article class="article">
+        <img class="article__thumbnail" src="images/edinburgh-damascus_thumbnail.png" alt="Road sign" />
```

Comment:
Good correction of image path and use of alt text

Reply:
Good correction of image path and use of alt text